zeebe/bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/builder/AbstractUserTaskBuilder.java

```diff
@@ -175,6 +176,14 @@ public B zeebeExternalFormReferenceExpression(final String expression) {
     return zeebeExternalFormReference(asZeebeExpression(expression));
   }
 
+  @Override
+  public B zeebeFormBindingType(final ZeebeBindingType bindingType) {
```

Comment:
FYI: Given that you can set a couple of other attributes that are not related to the form, I named this method `zeebeFormBindingType` instead of just `zeebeBindingType`.

Reply:
💭 It would be good to [document such intentions in the commit message](https://github.com/camunda/camunda/wiki/Pull-Requests-and-Code-Reviews#ive-written-a-clear-changelist-description) as well. This makes it easier to re-discover such decisions later, as the git commit message are available locally close to the code in most editors compared to GitHub PR comments which exist on the web. [Splitting up your commit into multiple smaller ones](https://github.com/camunda/camunda/wiki/Pull-Requests-and-Code-Reviews#ive-narrowly-scoped-my-changes) can help highlight changes for which a reason should be documented.

Reply:
Thanks, good points! 👍🏻 We're not really using the commit message body in my team, so I still need to get used to providing more information there ;) I've also realized that I could have split up my commit into multiple smaller ones. From this repo's history I see that the PRs often contain relatively small individual commits.

I will keep the points in mind for the next PR. Or was your comment meant as a suggestion to rewrite the commit history?

Reply:
@jfriedenstab My intention was simply to point it out as something for future changes. However, if you think it's easy to change then please take it along.

Reply:
It was relatively easy, so I split the commit into two.